.github/workflows/actions.yaml

```diff
@@ -21,13 +21,49 @@ jobs:
     steps:
       - name: Checkout csi-powermax
         uses: actions/checkout@v4
+        with:
+          path: "csi-powermax"
+      - name: Checkout dependent module gocsi
+        uses: actions/checkout@v4
+        with:
+          repository: dell/gocsi
+          path: 'gocsi'
```

Comment:
@bpjain2004 I think we don't need other libs/repo for UT. As I deleted other repos from my setup and it worked for me only with gopowermax.
<img width="496" alt="image" src="https://github.com/user-attachments/assets/bb218b3e-1e7b-48a6-9abc-bd89e7caf226">

<img width="722" alt="image" src="https://github.com/user-attachments/assets/8291c127-9dec-49b5-8c6b-f6aa2c28fcfa">

Reply:
thank you, removed cloning other repos.